### DISCUSSION TITLE
Path to Library Folder on root?

### DISCUSSION THREAD

**Discussion by Dirk71** [fsantini/KoboCloud#146] [Q&A] [answered]
Is it possible to load the books to **/my-folder** instead of **/.add/kobocloud/Library**? If so, how?

**Comment by fsantini** [comment, marked as answer]
You can try changing the [config_kobo.sh](https://github.com/fsantini/KoboCloud/blob/master/src/usr/local/kobocloud/config_kobo.sh) file. It might work, but it might not autoload the books.

**Reply by Dirk71** [reply]
Yes, I have tried but does not work.
